apps/vanilla/src/pages/2/src/views/roll-button-view.ts

```diff
@@ -0,0 +1,49 @@
+import { View } from './view';
+
+/** Events. */
+export interface ListenerEvents {
```

Comment:
```suggestion
interface ListenerEvents {
```
The interface is only used in this file